radis/lbl/loader.py

```diff
@@ -1289,6 +1289,11 @@ def _check_database_params(
         # Check input types are correct
         if isinstance(path, string_types):  # make it a list
             path = get_files_from_regex(path)
+            filtered_path = [fname for fname in path]
```

Comment:
leave a small comment it for why we're doing that (maybe reference the Issue too)